docs/paper/dev/getting-started/assisted-project-setup.md

```diff
@@ -0,0 +1,68 @@
+---
+slug: /dev/assisted-project-setup
+description: Step-by-step instructions on how to setup a plugin with the MinecraftDev plugin.
+---
+
+# Paper Project setup with the MinecraftDev IntelliJ plugin
+If you are new to plugin development you might be a little overwhelmed by the amount of things you need to do to get started.
+This guide aims to help you get started with setting up a new project using the MinecraftDev IntelliJ plugin.
+
+:::note
+
+This tutorial only works with IntelliJ IDEA. If you are using another IDE, please follow the [manual project setup guide](./manual-project-setup).
+
+:::
+# Installing the MinecraftDev plugin
+The first thing you need to do is install the [Minecraft Development](https://plugins.jetbrains.com/plugin/8327-minecraft-development) plugin. 
+You can do this by going to `File > Settings > Plugins` and searching for `Minecraft Development` under the `Marketplace` section.
+
+![](./assets/installing_plugin.png)
+
+Once you have installed the plugin, you will need to restart IntelliJ.
+To do that you can click the `Restart IDE` button that appears after installing the plugin.
+
+![](./assets/restart_ide.png)
+
+# Creating a new project
+Once you have installed the plugin, you can create a new project by going to `File > New > Project...` and selecting `Minecraft` from the list of options.
+
+![](./assets/new_project.png)
+
+You will be asked to provide some information about your project.
+* Name: The name of your project.
+* Location: The location of your project. This is where the project files will be stored.
+* Platform Type: The platform type you are developing for. This should be `Plugin`.
+* Platform: The platform you are developing for. This should be `Paper`.
+* Minecraft Version: The version of Minecraft you are developing for.
+* Plugin Name: The name of your plugin.
+* Main Class: The main class of your plugin. This should be the class that extends `JavaPlugin`.
+* Optional Settings: Here you can define things like authors, website, description, etc. These are optional and not required for the plugin to work.
+* Build System: The build system you want to use. Paper recommends using Gradle but you can use Maven if you prefer.
+* Paper manifest: Whether you want to use the new Paper Plugins or not. For now this is not recommended as it is still in development.
+* Group ID: The group ID of your project. This is used for Maven and Gradle. This is usually your domain name in reverse. If you don't know what you should put here, you can use something like `com.<yourname>`
```

Comment:
You should use a domain you own, so something like `io.github.<username>.<project>` or `me.<username>` is also commonly used

Reply:
but you most likely don't own me.<name> and not everybody has a GitHub account. I don't think this is to change.